src/server/svr_movejob.c

```diff
@@ -554,12 +553,12 @@ send_job_exec(job *jobp, pbs_net_t hostaddr, int port, struct batch_request *req
 	(void) strcpy(job_id, jobp->ji_qs.ji_jobid);
 
 	pqjatr = &((svrattrl *) GET_NEXT(attrl))->al_atopl;
-	jobid = PBSD_queuejob(stream, jobp->ji_qs.ji_jobid, destin, pqjatr, NULL, rpp, &msgid);
+	jobid = PBSD_queuejob(stream, jobp->ji_qs.ji_jobid, destin, pqjatr, NULL, PROT_TPP, &msgid);
```

Comment:
Do we call this function (and the others in this function) with anything but PROT_TPP?  If not, can we remove the argument?

Reply:
We call PBSD_queuejob and others funcs with PROT_TCP from client-side/IFL calls
for example: see __pbs_submit() in pbsD_submit.c which calls PBSD_queuejob() with 0 as prot value.
(I realized that I have not changed 0 to PROT_TCP in IFL calls, will do in next commit)